.github/workflows/ci.yaml

```diff
@@ -42,14 +42,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-version: [ "v1.19.16", "v1.22.0", "v1.25.3" ]
+        kubernetes-version: [ "1.23.17", "1.24.17", "1.25.13", "1.26.8", "1.27.5" ]
         cri: [ docker, containerd ]
         exclude:
-          - kubernetes-version: v1.19.16
-            cri: containerd
-          - kubernetes-version: v1.22.0
-            cri: containerd
-          - kubernetes-version: v1.25.3
+          - kubernetes-version: v1.25.13
             cri: docker
```

Comment:
@xqi-nr please be aware that we do not need to test docker as a CRI, [Kubernetes support has been removed](https://kubernetes.io/blog/2022/05/03/dockershim-historical-context/).


Thus, the `exclude` field can be removed and `cri` should be updated to `containerd` only

Also, can you please update [this reference to `kubectl` to the latest supported version](https://github.com/newrelic/newrelic-k8s-metrics-adapter/actions/runs/6124755647/workflow?pr=191#L57)?